plugins/org.jkiss.dbeaver.data.transfer.ui/src/org/jkiss/dbeaver/tools/transfer/ui/internal/DTUIMessages.java

```diff
@@ -172,6 +172,9 @@ public class DTUIMessages extends NLS {
 	public static String database_producer_page_input_objects_title_assign_error;
 	public static String database_producer_page_input_objects_message_error_reading_container_objects;
 	public static String database_producer_page_input_objects_error_message_auto_assign_failed;
+	public static String stream_producer_column_mapping_error_title;
```

Comment:
Please, fix tabs and incorrect indentation level (should be 4) in your strings in this file.